src/Illuminate/Http/Request.php

```diff
@@ -744,7 +744,8 @@ public function offsetExists($offset)
 	 */
 	public function offsetGet($offset)
 	{
-		return $this->input($offset);
+		$all = $this->all();
+		return $all[$offset];
```

Comment:
But it can cause an undefined index warning, which is not like any other accessor in Laravel.